src/libpriv/rpmostree-kargs-process.c

```diff
@@ -0,0 +1,331 @@
+/*
```

Comment:
Note: This file is copied straight from https://github.com/ostreedev/ostree/blob/master/src/libostree/ostree-kernel-args.c, just for testing the functionality purposes. Need to decide whether or not we should make the ostree-kernel-args public.

Reply:
Also Note: Functions I newly added will be commented with `Note: this function is newly added to the API` on top of the function. The other functions without those are from the old existing api.